### PR TITLE
Feature/rkm66 88 walk rally check in check out e stamp

### PR DESCRIPTION
**Pull request by D33102**
Add APIs and redesigned e-stamps...

### CODE REVIEW

File: src/pages/scan.tsx [hunk 1]
```diff
@@ -1,25 +1,45 @@
...
+    const { isAuthenticated, isReady } = useAuth();
     const [data, setData] = useState('');
     const [showModal, setShowModal] = useState(false);
-
-    const handleScanResult = (result: any, error: any) => {
-        if (result) {
-            setData(result.text);
+    const toast = useToast();
+    const router = useRouter();
+    const checkIn = async (token: any) => {
```
Comment (boomchanotai) [inline src/pages/scan.tsx]: after scan you should setData("") so, It will not be looped.

**Review by boomchanotai — CHANGES_REQUESTED** [review]
Super Cool! change a little bit

File: src/pages/scan.tsx (hunk 1 above)
Comment (D33102) [reply src/pages/scan.tsx]: @boomchanotai done :D

**Review by boomchanotai — APPROVED** [review]
Perfect!

**Pull request merged**